src/plugins/ardura/Subhoofer/2.2.1.yaml

```diff
@@ -0,0 +1,25 @@
+---
+name: Scrollscope
+author: Ardura
+homepage: https://github.com/ardura/Scrollscope
+description: A simple scrolling oscilloscope in rust with multichannel inputs and an analyzer
+date: 2024-07-12T07:00:00.000Z
+license: GPL3
+tags:
+  - Effect
```

Comment:
Same comment for more tags

Reply:
I added distortion because technically it adds some saturation in its effect